.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -0,0 +1,43 @@
+---
+name: 'Bug Report'
+about: 'Report a bug, something does not work as it supposed to'
+title: '[bug] SHORT DESCRIPTION'
+labels: 'type: bug'
+---
+
+<!--
+  Please don't forget to update the issue title.
+  Include all applicable information to help us reproduce your problem.
+
+  To help us debug your issue please explain:
+-->
+
+### Environment Details (include every applicable attribute)
+  * Operating System+version:
+  * Compiler+version:
+  * Conan version:
+  * Python version:
+
+### Steps to reproduce (Include if Applicable)
+
+
+### Logs (Executed commands with output) (Include/Attach if Applicable)
+
+<!--
+  Your log content should be related to the bug description, it can be:
+  - Conan command output
+  - Server output (Artifactory, conan_server)
+
+  To increase the logging level, you can configure the environment variable:
+    https://docs.conan.io/en/latest/reference/env_vars.html#conan-logging-level
+  Also, you can configure Conan logs if you need:
+    https://docs.conan.io/en/latest/mastering/logging.html
+-->
+
+<details><summary>Click to expand log</summary>
+
+```
+Put your log output here.
+```
+
+</details>
```

Comment:
Maybe I would remove the option to expand

Reply:
done